src/test/resources/application-localstack.properties

```diff
@@ -1,22 +1,28 @@
 
+sqs.provider=localstack-embedded
 sqs.rpc.queue.url=http://localhost:4576/queue/risk_profiler_change
 sqs.events.queue.name=events
 sqs.events.dlq.queue.name=events-dl
 
 sqs.events.endpoint.url=http://localhost:4576
+sqs.events.endpoint.region=eu-west-2
+sqs.events.aws.access.key.id=
+sqs.events.aws.secret.access.key=
 
-sqs.provider=localstack-embedded
+sqs.events.aws.dlq.access.key.id=
+sqs.events.aws.dlq.secret.access.key=
 
 ocgm.delay=0
 ocg.delay=0
 
 s3.provider=localstack-embedded
 s3.endpoint.region=eu-west-2
+endpoint.url=http://localhost:4572
```

Comment:
should this one be `s3.endpoint.url` to tie up with https://github.com/ministryofjustice/offender-risk-profiler/pull/336/files#diff-4f572a825d9c1ba641c39015afb06293d33d9e04f262fa855ba3283641ae79e2R113

Reply:
yes. it should